packages/vite/src/node/index.ts

```diff
@@ -4,6 +4,7 @@ export * from './optimizer'
 export * from './config'
 export { send } from './server/send'
 export { createLogger } from './logger'
+export { resolvePackageData, resolvePackageEntry } from './plugins/resolve'
```

Comment:
`PackageData` interface needs to be exposed as well (need a separate `export type` statement)